modules/creating-a-compliance-scan-schedule.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * manage-compliance/managing-compliance-20.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="creating-a-compliance-scan-schedule_{context}"]
+= Creating a compliance scan schedule
+
+By creating a compliance scan schedule, you can customize and automate your compliance scans to align with your operational requirements.
+
+.Procedure
+
+. In the {product-title-short} portal, navigate to the *Compliance (2.0) -> Cluster Compliance -> Schedules* tab.
+. Click *Create scan schedule*.
+. In the *Configuration options* page, provide the following information:
+** *Name*: Enter a name to easily identify and distinguish different compliance scans.
+** *Description*: Add a description to understand the reason for each compliance scan.
+** *Configure schedule*: Adjust the scan schedule to fit your desired schedule:
+*** *Frequency*: From the drop-down list, select how often you want to perform the scan. Frequencies include `Daily`, `Weekly`, and `Monthly`.
+*** *On day(s)*: From the list, select one or more days of the week on which you want to perform the scan. Valid values include `Monday`, `Tuesday`, `Wednesday`, `Thursday`, `Friday`, `Saturday`, `Sunday`, `The first of the month`, and `The middle of the month`.
++
+[NOTE]
+====
+These values are only applicable if you specify the frequency of scan as `Weekly` or `Monthly`.
+====
+
+*** *Time*: Start to type the time in `hh:mm` at which you want to execute the scan. From the list that is displayed, select a time.
+. Click *Next*.
+. In the *Clusters* page, select one or more clusters that you want to include in the scan.
+. Click *Next*.
+. In the *Profiles* page, select one or more profiles that you want to include in the scan. Profiles include `CIS OpenShift`, `PCI DSS OpenShift`, `NERC CIP RHCOS`, and `STIG RHCOS`.
```

Comment:
```suggestion
. In the *Profiles* page, select one or more profiles that you want to include in the scan.
```

The profiles there were just hard coded for testing.  The actual list will be anything provided by the compliance operator.